blocks/header/header.css

```diff
@@ -1,12 +1,13 @@
-
-
+/* stylelint-disable no-descending-specificity */
```

Comment:
Disabled the descending specificity style lint rule since it would have forced me to structure my CSS in what seemed like a super counter intuitive way.  I currently have the properties grouped together based on class names associated with elements that groups them into functions (section nav bar, subsection, tool menu, etc).  The rule however doesn't take that class name into account when calculating the specificity of the selector.  So I'm opting to just disable the rule for this file.